packages/wrapped-keys/src/lib/litActions/solana/src/signMessageWithSolanaEncryptedKey.js

```diff
@@ -0,0 +1,70 @@
+const { Keypair } = require('@solana/web3.js');
+const bs58 = require('bs58');
+const nacl = require('tweetnacl');
+
+(async () => {
+  const LIT_PREFIX = 'lit_';
+
+  let decryptedPrivateKey;
+  try {
+    decryptedPrivateKey = await Lit.Actions.decryptToSingleNode({
+      accessControlConditions,
+      chain: 'ethereum',
+      ciphertext,
+      dataToEncryptHash,
+      authSig: null,
+    });
+  } catch (error) {
+    Lit.Actions.setResponse({
+      response: `Error decrypting data to private key: ${error.message}`,
```

Comment:
Standardizing this since the tests depend on checking the extract string including the ":"
```suggestion
      response: `Error: When decrypting data to private key: ${error.message}`,
```